action/action.info.yml

```
@@ -0,0 +1,5 @@
+name: action
+type: module
+description: Dummy module to replace action.
+package: Useless Machines
+core_version_requirement: ^10 || ^11
```

```suggestion
core_version_requirement: ^11
```

you mean this? @svicervlad

---

And in composer.json